packages/firestore/test/unit/specs/persistence_spec.test.ts

```diff
@@ -197,7 +197,7 @@ describeSpec('Persistence:', ['persistence'], () => {
       .shutdown()
       .expectPrimaryState(false)
       .client(1)
-      .restart()
+      .tryAcquirePrimaryLease()
```

Comment:
Once https://github.com/firebase/firebase-js-sdk/pull/412 lands, you can merge it and replace this with

```
.runTimer(TimerId.ClientStateRefresh)
```

Can you add a TODO somewhere?

Reply:
Added TODO to spec_builder.ts's tryAcquirePrimaryLease.